### ISSUE TITLE
Sim CTD after update to stable V. 0.6.0

### ISSUE THREAD

**Issue by Jack737800** [flybywiresim/a32nx#4461]
**Mod Version**

```json
V. 0.6.0 - stable, via the installer
```

**Describe the bug**

Sim crashes back to desktop, while in loading screen.
When deleting FBW mod. from the community folder, Sim starts without CTD.

Installed WU4, Sim is V. 1.15.7.0
Yestday the Sim worked without any issues.

**To Reproduce**

1. Install stable FBW V. 0.6.0 via installer
2. Start the Sim 
3. Wait until CTD on the loading screen

Was this working before/when did the issue start occurring?

Is this a problem in the vanilla unmodded game?

Discord username (if different from GitHub):
X_TheOne_X1 #2622

**Comment by Benjozork** [comment]
Make sure to try the A32NX without anything else in the community folder.

**Comment by lautapa79** [comment]
I have the same problem, with the installer I download the new stable version and FS2020 closes on the loading screen. If I remove the a320 folder from the Community directory, FS2020 starts up without problems.

**Comment by Benjozork** [comment]
Please read my above message.

**Comment by lautapa79** [comment]
Taking everything out of the community folder works.

**Comment by Benjozork** [comment]
Not everything. Everything except the A32NX.

**Comment by lautapa79** [comment]
Sure, just leaving the a32nx folder

**Comment by CptRPeralta93** [comment]
> Make sure to try the A32NX without anything else in the community folder.

I´ve done that and works, the minute i put just 1 livery on the community folder, i cant open the game again.. this never hapen to me before.

**Comment by Benjozork** [comment]
Can you send the particular livery?

Keep in mind we do not make the livery and thus it's not our product

**Comment by CptRPeralta93** [comment]
yes i can understand that. 

The livery i put in the community folder, just to test was this one:

https://flightsim.to/file/7390/hifly-a320-neo-8k

I read some about that after some recent updates on the A320nx the previous liveries wont work anymore, so they give me this link

**Comment by AdamNZ** [comment]
That's not the A32NX FBW compatible one:
" A32NX: Unless otherwise stated by the author, these liveries may not work with the new A32NX Development branch as of today.
"

**Comment by CptRPeralta93** [comment]
ok, the FlybyWire team recomend any site with liveries?

**Comment by AdamNZ** [comment]
Flightsim.to is THE place to go. Just make sure you're in the dedicated FBW liveries category:
https://flightsim.to/c/liveries/flybywire-a32nx/

**Comment by Benjozork** [comment]
@CptRPeralta93 The livery is not for the A32NX, but my sim works with it loaded.

**Comment by CptRPeralta93** [comment]
> @CptRPeralta93 The livery is not for the A32NX, but my sim works with it loaded.

I've just empty the community folder again, put the A320nx folder back on and another livery and now I can finally fly. In what liveries concerned, I will try to look for FBW liveries category. Thank you @Benjozork and @AdamNZ for your help.

**Comment by Jack737800** [comment]
I can confirm that after deleting all liveries for the A32X, the sim loads up fine.
The installer showed me "uncompatible" liveries, so I converted them with the tool. I figured out that after that both liveries are in the community folder! The "old" ones which got converted weren´t deleted automatic (wasn´t aware of that - maybe worth to put a little reminder to do so.)
Anyhow, the liveries were downloaded from flightsim.to, but I´ll check now the above link t get some hopefully compatible ones.

Thanks
